pom.xml

```diff
@@ -153,8 +153,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>11</source>
-                    <target>11</target>
+                    <source>15</source>
```

Comment:
Why?

Reply:
look at the search charity query

Reply:
@Query("""
           select c from Charity c
           where (lower(c.name) like lower(concat(:text,'%')) or :text is null)
           and (lower(c.condition) like lower(concat(:condition,'%')) or :condition is null)
           and (lower(c.category.name) like lower(concat(:category,'%')) or :category is null)
           and (lower(c.subCategory.name) like lower(concat(:subCategory,'%')) or :subCategory is null)
           and c.user.email not like concat(:email,'%')
           """)
    List<Charity> searchCharity(@Param("text") String text,
                                @Param("condition") String condition,
                                @Param("category") String category,
                                @Param("subCategory") String subCategory,
                                String email);
   it healps easily write query and make comfortable and does not affect the other

Reply:
Ok